rocky/assets/css/components/style.scss

```diff
@@ -0,0 +1,82 @@
+dialog {
```

Comment:
I would rename this file, since it's not bundled with the modal/dialog component in this PR the "style" name is a bit weird and confusing, since it's not very specific. I would suggest naming it either dialog.scss since that's the HTML element it's targeting.